doc/phpactor.txt

```diff
@@ -204,9 +208,16 @@ are available for you to copy:
     au FileType php nmap <buffer> <Leader>K :PhpactorHover<CR>
     au FileType php nmap <buffer> <Leader>tt :PhpactorTransform<CR>
     au FileType php nmap <buffer> <Leader>cc :PhpactorClassNew<CR>
-    au FileType php nmap <buffer> <silent> <Leader>ee :PhpactorExtractExpression<CR>
-    au FileType php vmap <buffer> <silent> <Leader>ee :<C-u>PhpactorExtractExpression<CR>
-    au FileType php vmap <buffer> <silent> <Leader>em :<C-u>PhpactorExtractMethod<CR>
+    au FileType php nmap <buffer> <Leader>ci :PhpactorClassInflect<CR>
+    au FileType php nmap <buffer> <Leader>fr :PhpactorFindReferences<CR>
+    au FileType php nmap <buffer> <Leader>mf :PhpactorMoveFile<CR>
+    au FileType php nmap <buffer> <Leader>cf :PhpactorCopyFile<CR>
+    au FileType php nmap <buffer> <silent> <Leader>ee
+    :PhpactorExtractExpression<CR>
+    au FileType php vmap <buffer> <silent> <Leader>ee
+    :<C-u>PhpactorExtractExpression<CR>
+    au FileType php vmap <buffer> <silent> <Leader>em
+    :<C-u>PhpactorExtractMethod<CR>
```

Comment:
@yaegassy did you fix the whitespace manually or is my `vimdoc` behaving differently?

Reply:
Created an issue in vimdoc: https://github.com/google/vimdoc/issues/111

Reply:
The whitespace has been removed manually.
I have never used `vimdoc`.

Reply:
When using `vimdoc`, it seems that 1 line is wrapped when it becomes more than `80 characters`.

Reply:
It was a temporary solution, but if I use `\` and limit it to 80 characters, it doesn't wrap in vimdoc either.

**ftplugin/php/mappings.vim**

```vim
"     au FileType php
"       \ nmap <buffer> <silent> <Leader>ee :PhpactorExtractExpression<CR>
"     au FileType php
"       \ vmap <buffer> <silent> <Leader>ee :<C-u>PhpactorExtractExpression<CR>
"     au FileType php
"       \ vmap <buffer> <silent> <Leader>em :<C-u>PhpactorExtractMethod<CR>
```

Reply:
Use I updated to use `\` :)